Doc/library/sqlite3.rst

```diff
@@ -748,6 +748,41 @@ Connection Objects
       .. versionadded:: 3.11
 
 
+   .. method:: serialize(*, name="main")
+
+      This method serializes a database into a :class:`bytes` object.  For an
+      ordinary on-disk database file, the serialization is just a copy of the
+      disk file.  For an in-memory database or a "temp" database, the
+      serialization is the same sequence of bytes which would be written to
+      disk if that database were backed up to disk.
+
+      *name* is the database to be serialized, and defaults to the main
+      database.
+
+      .. note::
+
+         This method is only available if the underlying SQLite library has the
+         serialize API.
+
+      .. versionadded:: 3.11
+
+
+   .. method:: deserialize(data, /, *, name="main")
+
+      This method causes the database connection to disconnect from database
+      *name*, and reopen *name* as an in-memory database based on the
+      serialization contained in *data*.  Deserialization will fail with
+      ``SQLITE_BUSY`` if the database is currently in a read transaction or is
```

Comment:
That's true at the C level, but for the Python-level documentation it will be more useful to say what exception it will throw.

Generally I noticed that the documentation is partially copied from SQLite's own docs. That should be fine because SQLite is in the public domain and obviously it's a good source for documentation about SQLite itself.

Reply:
> Generally I noticed that the documentation is partially copied from SQLite's own docs. That should be fine because SQLite is in the public domain and obviously it's a good source for documentation about SQLite itself.

Yes, your observation is correct.

> That's true at the C level, but for the Python-level documentation it will be more useful to say what exception it will throw.

Yes, I'll reword it.